sub/source/docs/sub-hardware.rst

```diff
@@ -0,0 +1,83 @@
+.. _sub-hardware:
+
+==================================
+Default Sub Hardware Configuration
+==================================
+
+While the Sub firmware is currently being generalized to allow more flexibility, such as RC control, re-arrangement of outputs and inputs, etc.,
+current code is targeted for a specific ROV configuration with the following hardware setup:
+
+- Control is via a surface tether from a ground control station via MAVLink commands to the Sub's autopilot telemetry port, usually employing a joystick and multiple buttons for mode changes and peripheral hardware control. Control functions and autopilot outputs for thrusters and peripheral hardware have specific defaults (see :ref:`Sub defaults <sub-defaults>` below)
+- A depth sensor (underwater barometer) is required.
+- Frame configuration is defaulted to Vectored with 6 thrusters.
+
+.. _sub-defaults:
+
+Sub Defaults
+============
+
+* Default :ref:`frame type <FRAME_TYPE>` is VECTORED
+* RC is disabled via :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` = 0 and :ref:`FLTMODE_CH<FLTMODE_CH>` = 0.
+
+MAVLink Overrides
+-----------------
+Since pilot directional control, normally provided by RC in other vehicles is not enabled by default. Sub uses RC overrides via MAVLink, and the channel mapping is fixed:
+
+===============  ===================
+Pilot control    RC channel override
+===============  ===================
+Roll             1
+Pitch            0
+Yaw              3
+Throttle(Depth)  2
+Forward          4
+Lateral          5
```

Comment:
```suggestion
Yaw (Turn)       3
Heave (Vertical) 2
Surge (Forward)  4
Sway (Lateral)   5
```
Adjustments to better match [the firmware naming](https://github.com/ArduPilot/ardupilot/blob/master/libraries/SRV_Channel/SRV_Channel.cpp#L114), which links to [ship movement terminology](https://en.wikipedia.org/wiki/Ship_motions).

Reply:
maybe put the nautical after the conventional...will change